recipes/libcarna-python/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "libcarna-python" %}
+{% set version = "0.2.0" %}
+{% set repo_url = "https://github.com/kostrykin/LibCarna-Python" %}
+{% set docs_url = "https://libcarna.readthedocs.io" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "{{ repo_url }}/archive/refs/tags/{{ version }}-with-skvideo.zip"
+  sha256: 47d28a0c9f5963ea591909547a93f1df064bb69a0197655d936755765302b2af
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=3.5, <=3.28.3
+    - eigen >=3.0.5
+    - libcarna ==3.4
+    - pybind11 <3
+    - libxcrypt  # [py < 311]
+  host:
+    - libgl-devel      # [linux]
+    - libegl-devel     # [linux]
+    - libopengl-devel  # [linux]
+    - libglu
+    - libcarna ==3.4
+    - python >=3.10
```

Host has a strict pin, run has a flexible pin, like https://github.com/conda-forge/staged-recipes/pull/29984/files#diff-0a488422896b0e238ed5467b6208b15cef6fedd871c649a09afa9d279c583647R19
```suggestion
    - python 3.10
```